rfcs/20201102-convert-verbose-into-benchdnn-problem/README.md

```diff
@@ -0,0 +1,50 @@
+# RFC: Convert DNNL_VERBOSE output into benchdnn problems
+
+## Motivation
+
+There are several requests from users and developers to enable DNNL_VERBOSE
+output be "runnable" from benchdnn. Such requests mostly consider the following
+scenario: there is some integration of oneDNN into some framework, and there is
+some topology to run. If validation shows performance or correctness regression,
+it is very hard and/or time consuming to narrow down the specific guilty layer,
+and those users usually send a log file with DNNL_VERBOSE output, which is
+really massive. Manual conversion of these outputs into benchdnn problems takes
+enormous amount of time, since the topology usually consists of 100+ layers. It
+is just inconvenient not to have some tool that allows to perform such
+conversion very fast.
+
+## Tool Requirements
+
+* The tool should accept either a single line or a text file with lines.
+* The output should be fully benchdnn-compatible.
+* The tool should be easy to use.
+
+## Proposal
+
+### Option 1 (recommended) - Python script.
+
+Use the power of Python to create such tool. It will take the input and produce
+benchdnn batch files separated by drivers. Script advantages:
```

Comment:
I think it is also useful to produce benchdnn inputs in the same order the were provided in verbose, otherwise someone will have to map benchdnn results back into framework verbose (it will be hard for a model with 100+ layers).

Reply:
I've made an additional comment about the mode you've proposed. Thanks!